src/open_api/mod.rs

```diff
@@ -34,18 +53,36 @@ pub async fn run(cache: Arc<Cache>) -> Result<()> {
     let svr = OpenApiService::new(apis::Apis, "Scroll Rollup Explorer", "2.0")
         .server(format!("{open_api_addr}/api"));
 
+    let tokio_metrics = TokioMetrics::new();
     let ui = svr.swagger_ui();
     let spec = svr.spec();
     let app = Route::new()
         .nest("/", ui)
-        .nest("/api", svr)
+        .at("/tokio_metrics", tokio_metrics.exporter())
```

Comment:
What are `/tokio_metrics` and `/metrics` used for?

Reply:
`/metrics` is the custom, tokio_metrics is tokio have degisned metrics.